ras_party/controllers/pending_survey_controller.py

```diff
@@ -64,91 +67,110 @@ def pending_share_create(business_id, survey_id, email_address, shared_by, batch
     :param session: db session
     :param batch_number: batch_number
     :type batch_number: uuid
+    :param is_transfer: True if the record is for transfer survey
+    :type is_transfer: bool
     :rtype: void
     """
-    pending_share = PendingShares(business_id=business_id, survey_id=survey_id, email_address=email_address,
-                                  shared_by=shared_by, batch_no=batch_number)
+    pending_share = PendingSurveys(business_id=business_id, survey_id=survey_id, email_address=email_address,
+                                   shared_by=shared_by, batch_no=batch_number, is_transfer=is_transfer)
     session.add(pending_share)
 
 
 @with_db_session
-def delete_pending_shares(session):
+def delete_pending_surveys(session):
     """
     Deletes all the existing pending shares which has passed expiration duration
```

Comment:
```suggestion
    Deletes all the existing pending surveys which have expired
```

Quick update given the new method naming

Reply:
👍